python/docs/source/examples/bluesky.py

```diff
@@ -7,11 +7,14 @@
 
 import asyncio
 import time
+
 import kaskada as kd
 import pyarrow as pa
-
-from atproto.firehose import AsyncFirehoseSubscribeReposClient, parse_subscribe_repos_message
 from atproto import CAR, AtUri, models
+from atproto.firehose import (
```

Comment:
Nox linting changed the examples. Seems good to apply our linting conventions here as well.